Formula/a/autossh.rb

```diff
@@ -48,6 +49,6 @@ def install
 @@ -23,4 +23,4 @@ fi
  #AUTOSSH_PATH=/usr/local/bin/ssh
  export AUTOSSH_POLL AUTOSSH_LOGFILE AUTOSSH_DEBUG AUTOSSH_PATH AUTOSSH_GATETIME AUTOSSH_PORT
- 
+
```

Comment:
This happens to me all the time from my editor too. Fine to merge as-is but check locally the patch still applies.

Reply:
Confirmed that I was still able to build from source locally with this change.

Reply:
As a side note, sometimes this may still break patches, so I set `files.trimTrailingWhitespace` to `false` for the `homebrew-core` workspace in VS Code.

Reply:
@ZhongRuoyu @p-linnane I think it's a good reason to move these patches with trailing whitespace into Homebrew/formula-patches. This happens too often to maintainers and contributors.